CONTRIBUTING.md

```diff
@@ -127,13 +127,10 @@ cd $CUXFILTER_HOME
 - Create the conda development environment `cuxfilter_dev`:
 ```bash
 # create the conda environment (assuming in base `cuxfilter` directory)
-conda env create --name cuxfilter_dev --file conda/environments/cuxfilter_dev_cuda10.0.yml
+conda env create --name cuxfilter_dev --file conda/environments/all*.yaml
```

Comment:
This will fail if the directory `conda/environments` contains multiple `*.yaml` files. I would prefer to name the file explicitly.
```suggestion
conda env create --name cuxfilter_dev --file conda/environments/all_cuda-115_arch-x86_64.yaml
```